lighthouse-core/config/default.js

```diff
@@ -199,6 +209,10 @@ module.exports = {
       title: 'Meta Tags Used Properly',
       description: 'These are opportunities to improve the user experience of your site.',
     },
+    'manual-a11y-checks': {
+      title: 'Manual checks to verify',
+      description: 'These checks address areas which an automated testing tool cannot cover. They do not affect your score but it\'s important that you verify them manually. Learn more in our guide on [conducting an accessibility review](https://developers.google.com/web/fundamentals/accessibility/how-to-review).'
```

Comment:
I think this is a little wordy, though i don't have any particular suggestions. Between this and the overall a11y description you want to take another pass?

Reply:
I chopped out the middle sentence if that helps?